orbit/cmd/desktop/desktop.go

```diff
@@ -247,6 +248,11 @@ func main() {
 		}()
 
 		if runtime.GOOS == "darwin" {
+			homedir, err := os.UserHomeDir()
+			if err != nil {
+				log.Fatal().Err(err).Msg("failed to get user's home directory")
+			}
+			mrw := migration.NewReadWriter(filepath.Join(homedir, "Library/Caches/com.fleetdm.orbit"), constant.MigrationFileName)
```

Comment:
could you make this whole thing a shared function or alternatively the directory a constant somewhere?